python/py-setproctitle/Portfile

```diff
@@ -4,7 +4,8 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-setproctitle
-version             1.1.10
+version             1.2.1
```

Comment:
this version (nor the previous for that matter) lists py27 anymore as supported; please restrict to the latest supported version for the py27 subport

Reply:
Seems like 1.1.10 is fine with 2.7.